README.md

```diff
@@ -1,39 +1,50 @@
 # AWS IoT Device SDK for Java v2
 
-This document provides information about the AWS IoT device SDK for Java V2. This SDK is built on the [AWS Common Runtime](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html)
+The AWS IoT Device SDK for Java v2 connects your Java applications and devices to the AWS IoT platform. Built on the AWS Common Runtime, it handles the complexities of secure communication, authentication, and device management so you can focus on your IoT solution.
 
-*__Jump To:__*
+**Supported Platforms**: Linux, Windows, macOS, [Android](./documents/ANDROID.md)
 
-* [Installation](#installation)
-* [Android](./documents/ANDROID.md)
+*__Topics:__*
+* [Features](#features)
+* [Using SDK](#using-sdk)
+* [Getting Started](#getting-started)
 * [Samples](samples)
-* [Mac-Only TLS Behavior](#mac-only-tls-behavior)
-* [Getting Help](#getting-help)
-* [FAQ](./documents/FAQ.md)
-* [API Docs](https://aws.github.io/aws-iot-device-sdk-java-v2/)
 * [MQTT5 User Guide](./documents/MQTT5_Userguide.md)
-* [Migration Guide from the AWS IoT SDK for Java v1](./documents/MIGRATION_GUIDE.md)
+* [Getting Help](#getting-help)
+* [Resources](#resources)
+
+## Features
+
+The primary purpose of the AWS IoT Device SDK for Java v2 is to simplify the process of connecting devices to AWS IoT Core and interacting with AWS IoT services on various platforms. The SDK provides:
+
```

Comment:
nvm, the link for the rendered version has a line here that seems to no longer exist =D